polytracker/src/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-find_package(LLVM 13 CONFIG REQUIRED)
+find_package(LLVM 12 CONFIG REQUIRED)
```

Comment:
OK, so we are using LLVM-12. I think you mentioned why at some point, but I can't remember. Was there some technical difficulty in using 13+?

Reply:
See comments above.